test/fuzz/basic.ml

```diff
@@ -336,3 +336,95 @@ let block_type env =
   let+ param_type = list (param env)
   and+ result_type = list val_type in
   Arg.Bt_raw (None, (param_type, result_type))
+
+let memory_size = const Memory_size
+
+let memory_grow = const Memory_grow
+
+let memory_copy = const Memory_copy
+
+let memory_fill = const Memory_fill
+
+let memory_init = const (Memory_init (Raw 0))
+
+let memarg =
+  let* offset = range 10 in
+  let+ align = range 10 in
```

Comment:
Why 10 ?

Reply:
I don't know if it's ok to generate unconstrained integers.
I chose small integers arbitrarily (10). I try to remove the constraint ...

Reply:
Could you add a TODO with a link to https://github.com/OCamlPro/owi/pull/28#discussion_r1212838601 ?

Reply:
This has been fixed but not referenced here, sorry.